### PR TITLE
iCubGenova09: Add FT sensor to right upper leg

### PR DESCRIPTION
**Pull request by mfussi66**
As per title, this PR adds the FT sensor on the right leg of iCub3 to reflect the addition on the real robot.

Its transformation relative to the root link is the following:
```
-0.866025  -0.5        0
-0.5        0.866025   0
 0          0         -1
```
Which corresponds to the transform in the CAD, and therefore it corresponds to the way it is mounted:
MicrosoftTeams-image

cc @S-Dafarra @pattacini

### CODE REVIEW

**Review by traversaro — APPROVED** [review]

**Review by Nicogene — COMMENTED** [review]
@mfussi66 thanks for the PR, the CI is failing because there is a mismatch between the expected number of FT and the actual number since you added two sensors :

`icub-model-test : odd number of F/T sensor found in the model`
Could you please fix the test accordingly?

**Comment by Nicogene** [comment]
Actually the problem is that the left leg sensor is missing:

https://github.com/robotology/icub-models-generator/blob/a57abd304d818a2199b6a0b097866777d455db13/tests/icub-model-test.cpp#L375-L379

**Comment by mfussi66** [comment]
> Actually the problem is that the left leg sensor is missing:

Yep, only the right one was mounted on the robot. I guess changing the test to check against zero should suffice.

**Comment by Nicogene** [comment]
> Yep @Nicogene , only the right one was mounted on the robot. I guess changing the test to check against zero should suffice.

Yes, then if it is intentional it is fine to change the test

**Comment by mfussi66** [comment]
Additionally, I think I'll need to change the check on the correct orientation of the sensor:
https://github.com/robotology/icub-models-generator/blob/a57abd304d818a2199b6a0b097866777d455db13/tests/icub-model-test.cpp#L440-L465

**Comment by GiulioRomualdi** [comment]
cc @isorrentino

**Comment by traversaro** [comment]
@mfussi66 are we waiting for something or can we merge?

**Comment by mfussi66** [comment]
> @mfussi66 are we waiting for something or can we merge?

I was waiting for the approval from @Nicogene, on my side I think we can merge.

**Review by Nicogene — APPROVED** [review]
LGTM !

**Comment by pattacini** [comment]
Shall we merge?

**Review by traversaro — APPROVED** [review]

**Pull request merged**
